style/javascript/README.md

```diff
@@ -0,0 +1,18 @@
+JavaScript
+==========
+
+[Sample](sample.js)
+
+* Prefer ES6 classes over library provided classes.
```

Comment:
But if I'm using Backbone, I'll be using "library-provided classes" no matter what, right? So does this mean "avoid backbone because it's not in ES6"?